### PR TITLE
Adjust C# output directories to be structured like Java.

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: gapic/api/artman_pubsub.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ common:
     staging:
       location: git@github.com:googleapis/api-client-staging.git
 csharp:
-  gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-pubsub
+  gapic_code_dir: ${REPOROOT}/artman/csharp/java/google-cloud-pubsub
```
Comment (ethanbao) [inline gapic/api/artman_pubsub.yaml]: csharp/java?

File: gapic/api/artman_longrunning.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ common:
     staging:
       location: git@github.com:googleapis/api-client-staging.git
 csharp:
-  gapic_code_dir: ${REPOROOT}/artman/output/gax-dotnet
+  gapic_code_dir: ${REPOROOT}/artman/output/csharp/google-longrunning
```
Comment (ethanbao) [inline gapic/api/artman_longrunning.yaml]: To confirm this is intended, right?

File: gapic/api/artman_pubsub.yaml (hunk 1 above)
Comment (jskeet) [inline gapic/api/artman_pubsub.yaml]: Humbug - will fix, thanks.

File: gapic/api/artman_longrunning.yaml (hunk 2 above)
Comment (jskeet) [inline gapic/api/artman_longrunning.yaml]: Correct. LRO isn't in GAX for .NET - it's a separate package.

File: gapic/api/artman_pubsub.yaml (hunk 1 above)
Comment (jskeet) [inline gapic/api/artman_pubsub.yaml]: (Done and amended the commit)

**Pull request merged**